apps/api_web/lib/api_web/swagger_helpers.ex

```diff
@@ -273,6 +273,8 @@ defmodule ApiWeb.SwaggerHelpers do
       """
       #{opts[:desc]}
 
+      Revenue status indicates whether or not the vehicle is accepting passengers.
```

Comment:
Suggested reword:

"Revenue status indicates whether or not the vehicle is in revenue service - i.e. whether or not a vehicle is accepting passengers."